code/game/atoms.dm

```diff
@@ -798,3 +798,6 @@ its easier to just keep the beam vertical.
 	blessed = 1
 
 /atom/proc/update_icon()
+
+/atom/proc/holomapAlwaysDraw()
+	return 1
```

Comment:
So it can get read by tile.loc here https://github.com/d3athrow/vgstation13/pull/11517/commits/7f6a0ffbe24a6b383d7b27fa053159123091a11d#diff-80ea06d9553d1386818fb1d28e388cd5R26

Do you think I should have rather declared var/area/A = tile.loc ?